src/Feature/Syntax/Feature_Syntax.re

```diff
@@ -95,3 +95,23 @@ let handleUpdate = (bufferUpdate: BufferUpdate.t, highlights: t) => {
     highlights,
   );
 };
+
+let update = (highlights: t, msg) =>
+  switch (msg) {
+  | TokensHighlighted(tokens) => setTokens(tokens, highlights)
+  | BufferUpdated(bu) when !bu.isFull => handleUpdate(bu, highlights)
```

Comment:
`bu` is ambiguous. It fits just as well for `Buffer` as `BufferUpdate`

```suggestion
  | BufferUpdated(update) when !update.isFull => handleUpdate(update, highlights)
```

Reply:
Thanks @glennsl ! First time using the suggestions feature... that's cool